src/app/core/components/score/score.component.scss

```diff
@@ -0,0 +1,34 @@
+.container {
+  display: flex;
+  flex-flow: column;
+  gap: 8px;
+  padding: 8px;
+  border-radius: 12px;
+  background-color: grey;
+  align-items: center;
+  justify-content: center;
+  height: 100px;
+  width: 100px;
+}
+
+.container.good {
```

Comment:
the BEM version of this would be `container--good`. A notation I sometimes like in situations like that.